selfdrive/test/test_updated.py

```diff
@@ -176,6 +177,13 @@ def _check_update_state(self, update_available):
     self.assertEqual(update == "1", update_available, f"UpdateAvailable: {repr(update)}")
     self.assertEqual(self._read_param("UpdateFailedCount"), "0")
 
+    if update_available:
+      # validate our simple markdown parser produces the same output as `markdown` from pip
+      release_notes = self._read_param("ReleaseNotes")
+      with open(os.path.join(self.basedir, "RELEASES.md")) as f:
+        r = f.read().split("\n\n", 1)[0]
+      self.assertEqual(markdown.markdown(r, tab_length=2), release_notes)
+
```

Comment:
I wasn't able to fully run these tests locally, but I'm pretty sure this check will work correctly. Are these tests only run on comma devices? I assume the `markdown` library won't be a problem here, since I think this file is the unittest @pd0wm was referring to in https://github.com/commaai/openpilot/pull/22768#issuecomment-956503933

Reply:
I'd put the test in a separate unit test that we only run in github actions. Make sure you put the `markdown` dependency under dev-packages.